addons/html_builder/static/src/plugins/image/image_grid_option_plugin.js

```diff
@@ -0,0 +1,45 @@
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+import { ImageGridOption } from "./image_grid_option";
+
+class ImageGridOptionPlugin extends Plugin {
+    static id = "imageGridOption";
+
+    resources = {
+        builder_options: [
+            {
+                OptionComponent: ImageGridOption,
+                selector: "img",
+            },
+        ],
```

Comment:
We might need a sequence here to put it just before the `animation`, but not sure the right number